### PR TITLE
cephfs-top: check the minimum compatible python version

### PR DESCRIPTION
**Pull request by joscollin**
This PR is just for testing the jenkins build.

EDIT: now ready for test build and merge.

### CODE REVIEW

**Comment by joscollin** [comment]
jenkins test windows

**Comment by joscollin** [comment]
jenkins test windows

**Comment by joscollin** [comment]
jenkins build docs

**Comment by joscollin** [comment]
jenkins test api

**Comment by joscollin** [comment]
jenkins test docs

**Comment by joscollin** [comment]
jenkins test make check arm64

**Comment by joscollin** [comment]
jenkins test make check arm64

**Review by vshankar — CHANGES_REQUESTED** [review]

File: src/tools/cephfs/top/CMakeLists.txt [hunk 1]
```diff
@@ -5,3 +5,7 @@ if(WITH_TESTS)
   include(AddCephTest)
   add_tox_test(cephfs-top)
 endif()
+
+set(MINIMUM_COMPATIBLE_VERSION 3.6.0)
+find_package(Python3 ${MINIMUM_COMPATIBLE_VERSION} REQUIRED
```
[the remark this answers is not in the document]
Comment (joscollin) [reply src/tools/cephfs/top/CMakeLists.txt:10]: Is that a stricter version check? As I checked, anything above `3.6.0` should pass this check. I'm not using `EXACT` here.

[the remark this answers is not in the document]
Comment (vshankar) [reply src/tools/cephfs/top/CMakeLists.txt:10]: Stricter as compared to the top-level CMake check for Python >3

[the remark this answers is not in the document]
Comment (joscollin) [reply src/tools/cephfs/top/CMakeLists.txt:10]: updated

**Comment by vshankar** [comment]
jenkins retest this please

**Comment by vshankar** [comment]
make check failure is

> ERROR: Cannot install -r requirements.txt (line 3) and kubernetes because these package versions have conflicting dependencies.

what broke that now? :/

**Comment by joscollin** [comment]
> make check failure is
> 
> > ERROR: Cannot install -r requirements.txt (line 3) and kubernetes because these package versions have conflicting dependencies.
> 
> what broke that now? :/

The same error comes for other PRs also, not specific to this one.

**Comment by joscollin** [comment]
jenkins test make check

**Comment by joscollin** [comment]
jenkins test make check

**Comment by vshankar** [comment]
Builds are fine in shaman: https://shaman.ceph.com/builds/ceph/wip-vshankar-testing-20230503.090416/

@petrutlucian94 @dmick PTAL at the changes.

**Comment by petrutlucian94** [comment]
looks good to me

**Pull request merged**